e2e/README.md

```diff
@@ -54,15 +54,25 @@ are run. e.g.
 go test -run TestObjectStorageSmokeSuite github.com/rook/rook/e2e/tests/smoke --rook_version=v3.0.1
 ```
 
-#### Run Tests on pre-installed/exisitng Rook
+#### Run Tests on pre-installed/existing Rook
 To run tests against a pre-installed rook set  ```--skip_install_rook=true``` . 
 Setting skip_install_rook to true will ignore all other flags and run tests without installing rook first.
 
 e.g.
 ```
-go test -run TestFileStorageSmokeSuite github.com/rook/rook/e2e/tests/smoke --skip_install_rook=true
+go test -run TestFileSystemSmokeSuite github.com/rook/rook/e2e/tests/smoke --skip_install_rook=true
 ```
-
+#### Run Longhaul Tests
+Using go test `-count` option, any tests can be repeated n number of times to simulate load or longhaul test. Although 
+any test can be converted to longhaul test, it's ideal to run integration tests under load for extended period to simulate 
+realworld scenarios. Also a new custom test flag `load_parallel_runs` is added to control the number of concurrent opertaions being performed.
+For example look at test `github.com/rook/rook/e2e/tests/block/k8s/longhaul/basicBlockonghaul_test.go`
```

Comment:
Instead of a path to the file, a link would be better. It might be:
For example, look at the [block long haul test](URL)